src/helpers/executionResultHelper.ts

```diff
@@ -142,3 +142,45 @@ export function getHeaderItemName(headerItem: Execution.IResultHeaderItem): stri
     }
     return "";
 }
+
+export function getAttributeHeadersInDimension(
+    dimensions: Execution.IResultDimension[],
+): Array<Execution.IAttributeHeader["attributeHeader"]> {
+    const attributeHeaders: Array<Execution.IAttributeHeader["attributeHeader"]> = [];
+
+    dimensions.forEach((dimension: Execution.IResultDimension) => {
+        dimension.headers.forEach(
+            (wrappedHeader: Execution.IMeasureGroupHeader | Execution.IAttributeHeader) => {
+                const headerType = Object.keys(wrappedHeader)[0];
```

Comment:
please add comment about this. (why index 0?)

Reply:
Fixed by another approach